scripts/score.py

```diff
@@ -1,31 +1,46 @@
 import argparse
+import logging
 import os
 import joblib
 import pandas as pd
 from house_package_saachi.data_preprocessing import preprocess_housing_data
 from house_package_saachi.model_scoring import score_model
 
+def setup_logger(log_level, log_path=None, no_console_log=False):
+    logger = logging.getLogger()
```

Comment:
do the same as mentioned above here as well.

Reply:
Did necessary changes